packages/@expo/cli/src/utils/env.ts

```diff
@@ -153,6 +153,14 @@ class Env {
   get EXPO_USE_CUSTOM_INSPECTOR_PROXY(): boolean {
     return boolish('EXPO_USE_CUSTOM_INSPECTOR_PROXY', false);
   }
+
+  /** Enable Expo Router typed router generation */
+  get EXPO_USE_TYPED_ROUTES() {
```

Comment:
Add to docs https://github.com/expo/expo/blob/main/docs/pages/workflow/expo-cli.mdx#environment-variables